lxd/network/driver_ovn.go

```diff
@@ -1,778 +0,0 @@
-package network
```

Comment:
And this one removes it, can you move things around so it never shows in this branch?